openstack/nova/templates/api-deployment.yaml

```diff
@@ -41,7 +41,7 @@ spec:
 {{- end }}
       containers:
         - name: nova-api
-          image: {{.Values.global.imageRegistry}}/{{.Values.global.image_namespace}}/ubuntu-source-nova-api:{{.Values.imageVersionNovaApi | default .Values.imageVersion | required "Please set nova.imageVersion or similar"}}
+          image: {{.Values.global.imageRegistry}}/{{.Values.global.image_namespace}}/ubuntu-source-nova-api:{{.Values.imageVersionNovaApi | default .Values.imageVersion  }}
```

Comment:
Please do not modify the other templates outside healthcheck and vcenter-operator